app/cisc/cisc.go

```diff
@@ -0,0 +1,312 @@
+package main
+
+import (
+	"os"
+
+	"encoding/hex"
+
+	"path"
+
+	"io/ioutil"
+	"strings"
+
+	"github.com/dedis/cothority/app/lib/config"
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/services/identity"
+	"gopkg.in/codegangsta/cli.v1"
+)
+
+/*
+Cisc is the Cisc Identity SkipChain to store information in a skipchain and
+being able to retrieve it.
+
+This is only one part of the system - the other part being the cothority that
+holds the skipchain and answers to requests from the cisc-binary.
+*/
+
+var configFile string
+var sshDir string
+var sshConfig string
```

Comment:
OK, I got rid of those. The `clientApp` would be more difficult, so I leave it for now.

Also I'm doing all testing through an external bash-script, so I'm not too dependant on having no global variables (beside the fact that they _are_ evil, of course).